panther_manager/include/panther_manager/behavior_tree_utils.hpp

```diff
@@ -93,28 +96,75 @@ inline std::string GetLoggerPrefix(const std::string & bt_node_name)
 namespace BT
 {
 /**
- * @brief Converts a string to a vector of integers.
+ * @brief Converts a string to a vector of float.
  *
  * @param str The string to convert.
- * @return std::vector<int> The vector of integers.
+ * @return std::vector<float> The vector of float.
  *
- * @throw BT::RuntimeError Throws when there is no input or cannot parse int.
+ * @throw BT::RuntimeError Throws when there is no input or cannot parse float.
  */
 template <>
-inline std::vector<int> convertFromString<std::vector<int>>(StringView str)
+inline std::vector<float> convertFromString<std::vector<float>>(StringView str)
```

Comment:
Actually do we event used that?

Moreover why have we implemented int before?
https://github.com/BehaviorTree/BehaviorTree.CPP/blob/4.5.x/src/basic_types.cpp#L177 

o.O

Reply:
Vector float is used in joy check, because axes is define as a float vector.

Probably someone do not check it is already implemented.